cirq/ops/common_gates_test.py

```diff
@@ -620,13 +620,12 @@ def test_iswap_str():
 
 
 def test_iswap_unitary():
-    cirq.testing.assert_allclose_up_to_global_phase(
-        cirq.unitary(cirq.ISWAP),
-        np.array([[1, 0, 0, 0],
-                  [0, 0, 1j, 0],
-                  [0, 1j, 0, 0],
-                  [0, 0, 0, 1]]),
-        atol=1e-8)
+    cirq.testing.assert_allclose_up_to_global_phase(cirq.unitary(cirq.ISWAP),
+                                                    np.array([[1, 0, 0, 0],
+                                                              [0, 0, -1j, 0],
```

Comment:
I think this is wrong. The definition of an iswap is to put i's along the anti-diagonal. I carefully looked into this and asked several people about it before putting this matrix into the test.

See this quantum stackexchange question: https://quantumcomputing.stackexchange.com/a/2596/119